apidef/oas/schema/x-tyk-gateway.md

```diff
@@ -357,195 +367,459 @@ Tyk classic API definition: `base_identity_provided_by`.
 
-**Field: `allowedAlgorithms` (`[]string`)**
-AllowedAlgorithms is the array of HMAC algorithms which are allowed. Tyk supports the following HMAC algorithms:
+**Field: `enabled` (`boolean`)**
+Enabled allow/disallow the usage of the domain.
 
-- `hmac-sha1`
-- `hmac-sha256`
-- `hmac-sha384`
-- `hmac-sha512`
+**Field: `name` (`string`)**
+Name is the name of the domain.
 
-and reads the value from algorithm header.
+# DetailedActivityLogs
 
-Tyk classic API definition: `hmac_allowed_algorithms`.
+DetailedActivityLogs holds the configuration related to recording detailed analytics.
 
-**Field: `allowedClockSkew` (`double`)**
-AllowedClockSkew is the amount of milliseconds that will be tolerated for clock skew. It is used against replay attacks.
-The default value is `0`, which deactivates clock skew checks.
+**Field: `enabled` (`boolean`)**
+Enabled enables/disables detailed activity logs.
 
-Tyk classic API definition: `hmac_allowed_clock_skew`.
 
+Tyk classic API definition: `enable_detailed_recording`.
 
-### **AuthSources**
+# Global
 
-**Field: `header` ([AuthSource](#authsource))**
-Header contains configurations for the header value auth source, it is enabled by default.
+Global holds configuration applies globally: CORS and caching.
 
-Tyk classic API definition: `auth_configs[x].header`.
+**Field: `pluginConfig` ([PluginConfig](#pluginconfig))**
+PluginConfig contains the configuration related custom plugin bundles/driver.
 
-**Field: `cookie` ([AuthSource](#authsource))**
-Cookie contains configurations for the cookie value auth source.
+**Field: `cors` ([CORS](#cors))**
+CORS contains the configuration related to cross origin resource sharing.
 
-Tyk classic API definition: `auth_configs[x].cookie`.
+Tyk classic API definition: `CORS`.
 
-**Field: `query` ([AuthSource](#authsource))**
-Query contains configurations for the query parameters auth source.
+**Field: `prePlugin` ([PrePlugin](#preplugin))**
+PrePlugin contains configuration related to custom pre-authentication plugin.
 
-Tyk classic API definition: `auth_configs[x].query`.
+Tyk classic API definition: `custom_middleware.pre`.
 
+**Field: `postAuthenticationPlugin` ([PostAuthenticationPlugin](#postauthenticationplugin))**
+PostAuthenticationPlugin contains configuration related to custom post authentication plugin.
 
-### **AuthSource**
+Tyk classic API definition: `custom_middleware.post_key_auth`.
 
-**Field: `enabled` (`boolean`)**
-Enabled enables the auth source.
+**Field: `postPlugin` ([PostPlugin](#postplugin))**
+PostPlugin contains configuration related to custom post plugin.
 
-Tyk classic API definition: `auth_configs[X].use_param/use_cookie`.
+Tyk classic API definition: `custom_middleware.post`.
 
-**Field: `name` (`string`)**
-Name is the name of the auth source.
+**Field: `responsePlugin` ([ResponsePlugin](#responseplugin))**
+ResponsePlugin contains configuration related to custom post plugin.
```

Comment:
```suggestion
ResponsePlugin contains configuration related to to the custom plugin that is run during processing of the response from the upstream service.
```

Reply:
This isn't accurate, reworded as:

> ResponsePlugin contains configuration related to the custom plugin that generates a response.

Reply:
I was wrong here, as per https://tyk.io/docs/plugins/plugin-types/response-plugins/, reverted to your version.